packages/language/test/fourslash/preprocessor/include/incafter-in-config.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+/// <reference path="../../framework.ts" />
+
+// @filename: .pliplugin/proc_grps.json
+//// {
+////     "pgroups": [
+////         {
+////             "name": "default",
+////             "compiler-options": ["INCAFTER(PROCESS(lib))"],
+////             "libs": [
+////                 "cpy"
+////             ],
+////             "include-extensions": [
+////                 ".pli"
+////             ],
+////             "implicit-builtins": []
+////         }
+////     ]
+//// }
+
+// @filename: cpy/lib.pli
+//// DECLARE LIB_VAR FIXED;
+
+// @filename: main.pli
+////*PROCESS;
+//// DCL TEST FIXED;
+
+preprocessor.expectTokens(`*PROCESS;
+  DCL TEST FIXED;
+`);
```

Comment:
Shouldn't this also contain `DECLARE LIB_VAR FIXED;`?

Reply:
🤦 yeah, I'll put up another PR shortly